### PR TITLE
changes to enhance CLI support

### PR DESCRIPTION
**Pull request by poke30744**
It is to support console applications better.

### CODE REVIEW

File: LEProc/Program.cs [hunk 1]
```diff
@@ -88,7 +89,14 @@ private static void Main(string[] args)
 
             try
             {
-                Args = args;
+                var argWithQuotesList = Regex.Matches(Environment.CommandLine, @"[\""].+?[\""]|[^ ]+")
```
Comment (xupefei) [inline LEProc/Program.cs:92]: Can you add a few comments here to explain the logic?

Comment (poke30744) [reply LEProc/Program.cs:92]: This change is to keep original quotes in the command line. 

If we directly use args[] to make the command line, _cmd1.exe /c _cmd2.exe "path with space" will become _cmd1.exe /c _cmd2.exe path with space and will cause issues.

Comment (xupefei) [reply LEProc/Program.cs:92]: Thanks! Could you write the sentence into the code?

Comment (poke30744) [reply LEProc/Program.cs:95]: Sure. I have added comments to the code. Please review.

Comment (xupefei) [reply LEProc/Program.cs:95]: LGTM!

**Comment by imba-tjd** [comment]
I read some code and I don't think there is a good way to solve this scenario. Just `won't fix` when the filename contains quotes.

The correct way is to store every argument into an array, just like `string[] args`. Once you transformed them into a space-delimted string, it will become a mess.

**Pull request merged**